### PR TITLE
feat: update object detection model and docs

### PR DESCRIPTION
**Pull request by vli11**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [x] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?
update object detection model and docs

## Issue this PR will close

close: #325 

## Test Instructions if applicable
1. start camera simulator
2. run PIPELINE_PROFILE="object_detection" RENDER_MODE=1 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0 --workload ovms

### CODE REVIEW

File: docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md [hunk 1]
```diff
@@ -1,74 +1,81 @@
 # Run Object Detection Pipeline with New Model
-For running object detection pipeline, we have defaulted the model to person_vehicle_bike_detection_2000. You can use different model to run object detection. Here are the steps:
+
+OpenVINO Model Server has [many ways to run inferencing pipeline](https://docs.openvino.ai/2023.1/ovms_docs_server_api.html):
+TensorFlow Serving gRPC API, KServe gRPC API, TensorFlow Serving REST API, KServe REST API and OVMS C API through OpenVINO model server (OVMS). For running object detection pipeline, it is based on KServe gRPC API method, we have defaulted the model to ssd_mobilenet_v1_coco. You can use different model to run object detection. Here are the steps:
```
Comment (NeethuES-intel) [inline docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md:4]:
```suggestion
TensorFlow Serving gRPC API, KServe gRPC API, TensorFlow Serving REST API, KServe REST API and OVMS C API through OpenVINO model server (OVMS). For running object detection pipeline, based on KServe gRPC API method, default model used is ssd_mobilenet_v1_coco. You can use different model to run object detection. Here are the steps:
```

File: docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md [hunk 2]
```diff
@@ -1,74 +1,81 @@
...
+## Build and Run Pipeline
 
-## Rebuild and Run Pipeline
-1. Rebuild the python app and profile-launcher: `make build-python-apps`
-2. Restart simulator camera if not started: `./camera-simulator/camera-simulator.sh`
-3. To start object detection pipeline: `PIPELINE_PROFILE="object_detection" RENDER_MODE=1 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0 --workload ovms`
\ No newline at end of file
+1. Build the python app and profile-launcher: `make build-python-apps`
+2. Download sample video files: `cd benchmark-scripts/ && ./download_sample_videos.sh && cd ..`
+3. Start simulator camera if not started: `./camera-simulator/camera-simulator.sh`
```
Comment (antoniomtz) [inline docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md:80]:
```suggestion
3. Start simulator camera if not started: `make run-camera-simulator`
```

**Review by antoniomtz — CHANGES_REQUESTED** [review]

File: docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md [hunk 3]
```diff
@@ -1,74 +1,81 @@
...
+      "latest": {
+        "num_versions": 1
+      }
+    }
 ```
-!!! Note
-    `shape` is optional and takes precedence over batch_size, please remove this attribute if you don't know the value for the model.
-
 !!! Note
     Please leave `target_device` value as it is, as the value `{target_device}` will be recognized and filled out by script run.
```
Comment (jim-wang-intel) [inline docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md:34]:
```suggestion
    Please leave `target_device` value as it is, as the value `{target_device}` will be recognized and replaced by script run.
```

File: docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md [hunk 4]
```diff
@@ -1,74 +1,81 @@
...
+    ├── 1
+    ├── checkpoint
+    ├── frozen_inference_graph.pb
+    ├── model.ckpt.data-00000-of-00001
+    ├── model.ckpt.index
+    ├── model.ckpt.meta
+    ├── pipeline.config
+    └── saved_model
+        ├── saved_model.pb
+        └── variables
```
Comment (jim-wang-intel) [inline docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md:58]: why this `ssd_mobilenet_v1_coco_2018_01_28` is a sub-directory of ssd_mobilenet_v1_coco? and do we need this folder?

File: docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md [hunk 5]
```diff
@@ -1,74 +1,69 @@
...
 
-## Rebuild and Run Pipeline
-1. Rebuild the python app and profile-launcher: `make build-python-apps`
-2. Restart simulator camera if not started: `./camera-simulator/camera-simulator.sh`
-3. To start object detection pipeline: `PIPELINE_PROFILE="object_detection" RENDER_MODE=1 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0 --workload ovms`
\ No newline at end of file
+1. Build the python app and profile-launcher: `make build-python-apps`
+2. Download sample video files: `cd benchmark-scripts/ && ./download_sample_videos.sh && cd ..`
+3. Start simulator camera if not started: `make run-camera-simulator`
+4. To start object detection pipeline: `PIPELINE_PROFILE="object_detection" RENDER_MODE=1 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0 --workload ovms`
```
Comment (jim-wang-intel) [inline docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md:69]:
```suggestion
4. To start object detection pipeline: `PIPELINE_PROFILE="object_detection" RENDER_MODE=1 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0 --workload ovms`
5. To stop the running pipelines: `make clean-profile-launcher` to stop and clean up the client side containers, or `make clean-all` to stop and clean up everything.
```

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Review by antoniomtz — APPROVED** [review]
LGTM

**Pull request merged**